distributed-commandbus-jgroups/src/main/java/org/axonframework/jgroups/commandhandling/JGroupsConnector.java

```diff
@@ -151,7 +151,7 @@ public void connect() throws Exception {
 
         Address localAddress = channel.getAddress();
         String localName = channel.getName(localAddress);
-        SimpleMember<Address> localMember = new SimpleMember<>(localName, localAddress, null);
+        SimpleMember<Address> localMember = new SimpleMember<>(localName, localAddress, true, null);
```

Comment:
It's a minor thing, but I'd say it would be nice if instead of `true`, you'd have a constants `LOCAL_MEMBER` in this constructor.